packages/prosemirror-lwdita-demo/cypress/e2e/image.cy.ts

```diff
@@ -0,0 +1,90 @@
+describe('handling images', () => {
+  beforeEach(() => {
+    window.localStorage.setItem('file', `<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD LIGHTWEIGHT DITA Topic//EN" "lw-topic.dtd">
+<topic id="program">
+  <title>Test File 2</title>
+  <body>
+    <section>
+      <p>A test paragraph.</p>
+    </section>
+  </body>
+</topic>`);
+  })
+
+  it('remote image using the toolbar button', () => {
+    cy.visit('http://localhost:1234/')
+    .get('#editor > div > div.ProseMirror > article > div > section > p')
+    .click()
+    .get('#editor > div > div.ProseMirror-menubar > span:nth-child(16) > div')
+    .click()
+    .get('#urlInput')
+    .type('https://static.evolvedbinary.com/petal/eb-rose-small.png')
+    .get('#okButton')
+    .click()
+    .get('figure img')
+    .should('have.attr', 'src', 'https://static.evolvedbinary.com/petal/eb-rose-small.png')
+  });
+
+  it('embeds remote image using the toolbar button', () => {
+    cy.visit('http://localhost:1234/')
+    .get('#editor > div > div.ProseMirror > article > div > section > p')
+    .click()
+    .get('#editor > div > div.ProseMirror-menubar > span:nth-child(16) > div')
```

Comment:
@marmoure This is a very "risky" selector, because you depend the test on the position of a generic element which might change whenever we add new toolbar elements, instead of directly addressing the element with the specific classname. `#editor .ProseMirror-menubar .ProseMirror-icon.ic-image` should be specific enough.

Reply:
@marmoure What is the status here?

Reply:
@marmoure Why didn't you choose my suggestion `#editor .ProseMirror-menubar .ProseMirror-icon.ic-image` and inserted `#editor > div > div.ProseMirror-menubar > span > div.ic-image` which is again very specific as you know rely on a very specific markup and nesting of elements inside. Just rely on the classnames, that's enough. 

As we will soon refactor the markup for the toolbar, this code needs to be updated as well, so in this case we can leave it for now.